AGPs/AGP-Comprehensive-Aragon-Bug-Bounty-Program.md

```diff
@@ -0,0 +1,56 @@
+---
+AGP: N/A
+Title: Comprehensive Aragon Bug Bounty Program
+Author: burrrata (@burrrata)
+Status: Stage III
+Track: Association
+Created: 2019-10-02
+---
+
+# AGP-X: Comprehensive Aragon Bug Bounty Program
+
+## Description of desired Association policy change
+
+This proposal is to create a comprehensive [Bug Bounty Program](https://wiki.aragon.org/dev/bug_bounty/) for all funded Aragon applications and tooling. 
+
+### Bug Bounty Scope
+
+The scope of the [Aragon Association Bug Bounty Program](https://wiki.aragon.org/dev/bug_bounty/) will be expanded to make all Nest grantees and Flock team applications eligible. Applications will only be accepted into the bug bounty program if they have undergone a security audit, incorporated all major and critical security audit recommendations, and shipped to mainnet. 
```

Comment:
We will need to increase the scope to include the Aragon Court soon, I would preemptively add it to the list even though the program won't launch until A1 considers de codebase ready for it.

Reply:
I believe that this would be covered as A1, a Flock team, is building the Aragon Court. Added a comment mentioning the Aragon Court to make it explicitly clear that it would be covered.